lib/checkout_sdk/risk/location.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module CheckoutSdk
+  module Risk
+    # @!attribute latitude
+    #   @return [String]
+    # @!attribute longitude
+    #   @return [String]
```

Comment:
This ones are defined as numbers in the api-reference. I don't know if it makes a difference or not